README.md

```diff
@@ -108,7 +108,7 @@ Identifying users allows you to target specific users from the [Flagsmith dashbo
 To check if a feature exists for a given user Identity:
 
 ```dart
-final user = FeatureUser(identifier: 'flagsmith_sample_user');
+final user = Identity(identifier: 'flagsmith_sample_user');
```

Comment:
This would be a breaking change to existing consumers of the SDK it looks like, need to make sure we roll a major version if this gets merged.

Is there a reason why this was changed? Identity is the terminology we use throughout Flagsmith

Reply:
exactly but current client uses FeatureUser for same model. so i refactored this model name to common naming a cross almost all other clients.